pkgs/os-specific/darwin/skhd/default.nix

```
@@ -26,6 +26,8 @@ stdenv.mkDerivation (finalAttrs: {
     "BUILD_PATH=$(out)/bin"
   ];
 
+  env.NIX_CFLAGS_COMPILE = lib.optionalString stdenv.isDarwin "-Wno-error=implicit-function-declaration";
```

No need to use `lib.optionalString` in a darwin-only package, I guess.

```suggestion
  env.NIX_CFLAGS_COMPILE = "-Wno-error=implicit-function-declaration";
```